packages/serverless/src/apollo/app/user.ts

```diff
@@ -152,9 +163,15 @@ export const typeDefsUser = gql`
     publicKey: String!
   }
 
+  input FindUser {
+    userName: String
+    email: String
```

Comment:
I think we can add searching by public key too

Reply:
Resolved